homeassistant/components/alexa/capabilities.py

```diff
@@ -614,3 +629,55 @@ def get_property(self, name):
             return None
 
         return {"value": temp, "scale": API_TEMP_UNITS[unit]}
+
+
+class AlexaSecurityPanelController(AlexaCapibility):
+    """Implements Alexa.SecurityPanelController.
+
+    https://developer.amazon.com/docs/device-apis/alexa-securitypanelcontroller.html
+    """
+
+    def __init__(self, hass, entity):
+        """Initialize the entity."""
+        super().__init__(entity)
+        self.hass = hass
+
+    def name(self):
+        """Return the Alexa API name of this interface."""
+        return "Alexa.SecurityPanelController"
+
+    def properties_supported(self):
+        """Return what properties this entity supports."""
+        return [{"name": "armState"}]
+
+    def properties_proactively_reported(self):
+        """Return True if properties asynchronously reported."""
+        return True
```

Comment:
For another PR, but I realize that we should probably make this return `alexa_config.should_report_state`. Not everyone could have state reporting enabled.